t/Util.pm

```diff
@@ -240,18 +242,39 @@ sub sig_num {
     firstidx { $_ eq $name } split " ", $Config::Config{sig_name};
 }
 
-# returns a hash containing `port`, `tls_port`, `guard`
+# returns a hash containing `port`, `tls_port`, `quic_port`, `guard`
+# Set { quic => { disable => 1} } to disable QUIC support.
 sub spawn_h2o {
     my ($conf) = @_;
     my @opts;
     my $max_ssl_version;
 
+    my $quic = (ref($conf) eq 'HASH' ? $conf->{quic} : undef) // {};
+
     # decide the port numbers
     my ($port, $tls_port) = empty_ports(2, { host => "0.0.0.0" });
-    my @all_ports = ($port, $tls_port);
+    my $quic_port;
+    my @all_ports = (
+        {
+            port => $port,
+            proto => 'tcp',
+        },
+        {
+            port => $tls_port,
+            proto => 'tcp',
+        },
+    );
+
+    if (!$quic->{disable}) {
```

Comment:
Do we need this option at all?

We do not have this for TLS, I wonder why it's necessary for QUIC.

Reply:
There's a session resumption setting that aborts if QUIC is enabled (https://github.com/h2o/h2o/blob/master/src/ssl.c#L1097), so I think this option is needed.